### DISCUSSION TITLE
Contract award notice - Withheld publication of information

### DISCUSSION THREAD

**Discussion by AnrivaDeloitte** [OP-TED/eForms-SDK#869] [Q&A] [answered]
Dear Publication Office,

while developing the contract award notice type 29, we have found out there are 176 fields that may remain non-public ("unpublished") for a defined period. As a result, the xml file produced for notice type 29 does not show this information.

Taking this into account, we would ask you:

-	How long will this information be withheld from publication?

-	Since ARIA S.p.A., on behalf of Lombardy Region (Italy), is implementing its own editor for eFORMS, is it mandatory to implement fields that may remain unpublished and their Business Rules?

-	How can ARIA S.p.A. submit this information to TED if it should be initially withheld from publication?

Thank you in advance for your answers and we look forward to hearing from you soon.
 
Kind regards
[fieldd privacy.txt](https://github.com/OP-TED/eForms-SDK/files/14720168/fieldd.privacy.txt)

**Comment by YvesJo** [comment, marked as answer]
Hi,
Withholding information publication is an option provided to the notice author, it is not mandatory and the submitted XML shall contain all the information. Publications Office internal systems will hide or display information based on the provided data (i.e. presence of the field BT-195(xxxx)-yyy).
The date when information will get published is driven by the field BT-198(xxx)-yyy; this field was initially mandatory when a field information publication had to be withheld and became optional with SDK 1.10.
When the field information should not be immediately published (i.e. when BT-195(xxx)-yyy exists) and BT-198(xxx)-yyy is specified, then the date in BT-198(xxx)-yyy is used by our systems to publish the information. 
When the field information should not be immediately published (i.e. when BT-195(xxx)-yyy exists) and BT-198(xxx)-yyy is not specified, then the data will never get public. 
For the field 198(xxx)-yyy, there is a rule that limits the possible date to 10 years after submission.
KR
